doc/classes/RenderingDevice.xml

```diff
@@ -918,6 +918,7 @@
 			<description>
 				Creates a new texture. It can be accessed with the RID that is returned.
 				Once finished with your RID, you will want to free the RID using the RenderingDevice's [method free_rid] method.
+				[b]Note:[/b] [param data] takes an [Array] of [PackedByteArray]s. For [constant TEXTURE_TYPE_1D], [constant TEXTURE_TYPE_2D], and [constant TEXTURE_TYPE_3D] types, this array should only have one element, a [PackedByteArray] containing all the data for the texture. For [code]_ARRAY[/code] and [code]_CUBE[/code] types, the length should be the same as the number of [member RDTextureFormat.array_layers] in [param format].
```

Comment:
Probably should not be a note because it pretty much describes how to use a parameter.
```suggestion
				[param format]'s [member RDTextureFormat.texture_type] determines what should be inserted in the [param data] array. For [constant TEXTURE_TYPE_1D], [constant TEXTURE_TYPE_2D], and [constant TEXTURE_TYPE_3D] types, this array should have only one element: a [PackedByteArray] containing all the data for the texture. For [code]_ARRAY[/code] and [code]_CUBE[/code] types, the length should be the same as the number of [member RDTextureFormat.array_layers] in [param format].
```
I'd also personally avoid things like "For [code]_ARRAY[/code] and [code]_CUBE[/code] types" because they're a bit weird to parse, but I have no suggestions.

Reply:
It was genuinely an improvement in wording, outside of the "note or not note" argument. 😢